crates/store/re_sorbet/src/lib.rs

```diff
@@ -66,3 +66,29 @@ pub enum BatchType {
     /// Potentially multiple entities
     Dataframe,
 }
+
+/// Get the chunk ID from the metadata of the Arrow schema
+/// of a record batch containing a sorbet chunk.
+///
+/// Returns one of:
+/// * `Ok`
+/// * [`SorbetError::MissingChunkId`]
+/// * [`SorbetError::ChunkIdDeserializationError`]
+// TODO(#10343): remove this
+pub fn chunk_id_of_schema(
+    schema: &arrow::datatypes::Schema,
+) -> Result<re_types_core::ChunkId, SorbetError> {
+    let metadata = schema.metadata();
+    if let Some(chunk_id_str) = metadata
+        .get("rerun:id")
+        .or_else(|| metadata.get("rerun.id"))
```

Comment:
Nit: maybe create a helper method in `v0_23`? That way we won't forget to remove this check someday.

I did the same thing with `is_component_column`.